README

```diff
@@ -22,56 +22,60 @@ list is available below.
 
 [source, console]
 ----
-$ sudo apt install libpam-u2f pamu2fcfg
+sudo apt install libpam-u2f pamu2fcfg
 ----
 
 Additionally, Yubico maintains a PPA with up-to-date versions of pam-u2f.
 
 [source, console]
 ----
-$ sudo apt install software-properties-common
-$ sudo apt-add-repository ppa:yubico/stable
-$ sudo apt update
-$ sudo apt install libpam-u2f pamu2fcfg
+sudo apt install software-properties-common
+sudo apt-add-repository ppa:yubico/stable
+sudo apt update
+sudo apt install libpam-u2f pamu2fcfg
 ----
 
 === Fedora
 
 [source, console]
 ----
-$ sudo dnf install pam-u2f pamu2fcfg
+sudo dnf install pam-u2f pamu2fcfg
 ----
 
 === macOS
 
 [source, console]
 ----
-$ brew install pam-u2f
+brew install pam-u2f
 ----
 
 [[building]]
 == Building from a source tarball
 
-This project uses 'autoconf', 'automake', 'pkg-config' and 'libtool'
-to achieve portability and ease of use.
+This project depends on https://developers.yubico.com/libfido2[`libfido2`]
+(>= 1.3.0).
+Versions of this project up to 1.0.8 used `libu2f-host` and
+`libu2f-server`.
 
-In addition, https://developers.yubico.com/libfido2['libfido2'] (>=
-1.3.0) is needed. Versions of this project up to 1.0.8 used
-`libu2f-host` and `libu2f-server`. On Ubuntu, the necessary dependencies can be
-installed using
+On Ubuntu, the dependencies needed to build the source tarball can be installed
+using:
 
 [source, console]
 ----
-# apt install --no-install-recommends autoconf automake libtool pkg-config \
-    libfido2-dev libpam-dev libssl-dev
+sudo apt install --no-install-recommends \
+    libfido2-dev \
+    libpam-dev \
+    libssl-dev \
+    pkg-config \
+;
```

Comment:
Splitting these to into the two sections make sense to me :+1: 

My one ask is that we drop the `;` (and consequently also the last `\`) from these blocks, I do very much appreciate the trick for minimizing diffs in scripts though!

(At some point in the future -- perhaps as part of the build system migration -- I think it'd make sense for us to just list the runtime and build dependencies without relating them to any particular distribution or package manager instruction.)

Reply:
> Splitting these to into the two sections makes sense to me 👍
> 
> My one ask is that we drop the `;` (and consequently also the last `\`) from these blocks, I do very much appreciate the trick for minimizing diffs in scripts though!

OK :)

> (At some point in the future -- perhaps as part of the build system migration -- I think it'd make sense for us to just list the runtime and build dependencies without relating them to any particular distribution or package manager instruction.)

Unfortunately, packages do not use the same name in all distros. E.g. ninja is called `ninja-build` under Debian, and just `ninja` under openSUSE. But I think we can find some decent arrangement. A problem for another day.

Overall, I agree.

Reply:
> Unfortunately, packages do not use the same name in all distros. E.g. ninja is called ninja-build under Debian, and just ninja under openSUSE. But I think we can find some decent arrangement. A problem for another day.

Ah, yes, sorry for being unclear.  My point is, for this exact reason (!), that we just list packages using their _own_ canonical names and letting people figure out their distribution's variants themselves ([example from libfido2](https://github.com/yubico/libfido2?tab=readme-ov-file#dependencies), [example from yubihsm-shell](https://github.com/yubico/yubihsm-shell?tab=readme-ov-file#dependencies)).  But let's discuss that separately :slightly_smiling_face: